Failing tests on NixOS: NotImplementedError: Can't perform this operation for unregistered loader type

Hey,
two tests are failing on NixOS, not sure about the error yet: `rnginline/test/test_rnginline.py` and `rnginline/test/test_cmdline.py`. The trace looks like this:
```
rnginline/test/test_rnginline.py:113: in <module>
    test_testcases_testcases = _load_testcases()
rnginline/test/test_rnginline.py:42: in _load_testcases
    assert not [f for _files in [[schema], positive_cases, negative_cases]
rnginline/test/test_rnginline.py:43: in <listcomp>
    for f in _files if not pr.resource_exists(__name__, f)]
/nix/store/zfski6pdfdz1wj2xjiqrkp1m3jdyi3ly-python3.10-setuptools-63.2.0/lib/python3.10/site-packages/pkg_resources/__init__.py:1144: in resource_exists
    return get_provider(package_or_requirement).has_resource(resource_name)
/nix/store/zfski6pdfdz1wj2xjiqrkp1m3jdyi3ly-python3.10-setuptools-63.2.0/lib/python3.10/site-packages/pkg_resources/__init__.py:1415: in has_resource
    return self._has(self._fn(self.module_path, resource_name))
/nix/store/zfski6pdfdz1wj2xjiqrkp1m3jdyi3ly-python3.10-setuptools-63.2.0/lib/python3.10/site-packages/pkg_resources/__init__.py:1482: in _has
    raise NotImplementedError(
E   NotImplementedError: Can't perform this operation for unregistered loader type
```
Best regards
Jonas

--- a/rnginline/test/test_rnginline.py
+++ b/rnginline/test/test_rnginline.py
@@ -40,7 +40,7 @@ def _load_testcases():
 
         assert positive_cases
         assert not [f for _files in [[schema], positive_cases, negative_cases]
-                    for f in _files if not pr.resource_exists(__name__, f)]
+                    for f in _files if not pr.resource_exists(TESTPKG, f)]
 
         for file in positive_cases:
             testcases.append((schema, file, True))

